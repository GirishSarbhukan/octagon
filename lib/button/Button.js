'use strict';

exports.__esModule = true;

var _Button = require('semantic-ui-react/dist/commonjs/elements/Button');

var _Button2 = _interopRequireDefault(_Button);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

exports['default'] = _Button2['default'];
module.exports = exports['default'];
//# sourceMappingURL=Button.js.map