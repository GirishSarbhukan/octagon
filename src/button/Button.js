import Button from 'semantic-ui-react/dist/commonjs/elements/Button'



export default Button
