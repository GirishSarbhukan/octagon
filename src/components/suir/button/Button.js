import Button from 'semantic-ui-react/dist/es/elements/Button'
export default Button
