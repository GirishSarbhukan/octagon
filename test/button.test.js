import React from 'react'
import Button from '../src/button/Button'
import renderer from 'react-test-renderer'

const { test, expect } = global

test('Button rendered', () => {
  const btn = renderer.create(<Button>TestButton</Button>)
  let tree = btn.toJSON()
  expect(tree).toMatchSnapshot()
})
