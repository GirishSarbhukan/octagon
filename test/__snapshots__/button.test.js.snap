exports[`test Button rendered 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}>
  TestButton
</button>
`;
